lib/mutant/result.rb

```diff
@@ -121,11 +121,7 @@ def stop?
 
     # Test result
     class Test
-      include Result, Anima.new(
-        :passed,
-        :runtime,
-        :tests
```

Comment:
That one was entirely unused. Historic artifact, but transitively referenced lots of objects that where needlessly run through the wire.